guides/source/action_controller_overview.md

```diff
@@ -1103,7 +1103,8 @@ You can set it to a String, a Regexp, or an array of both.
 config.filter_redirect.concat ['s3.amazonaws.com', /private_path/]
 ```
 
-Matching URLs will be marked as '[FILTERED]'.
+Matching URLs will be replaced with '[FILTERED]'. However, if you wish to only the filter parameters, not the whole URLs,
```

Comment:
The second sentence should be reworded to `However, if you wish to only filter **the** parameters, not the whole URLs,`